static/app/views/issueDetails/actions/index.tsx

```diff
@@ -380,7 +376,7 @@ export function Actions(props: Props) {
           size: 'sm',
         }}
         items={[
-          ...(isIgnored || hasEscalatingIssues
+          ...(isIgnored
```

Comment:
Can we remove this whole block? `isIgnored || hasEscalatingIssues` -> `isIgnored || True` -> `True`?